src/lib/y2partitioner/actions/controllers/fstabs.rb

```diff
@@ -332,35 +332,34 @@ def setup_blk_filesystem(filesystem)
         #
         # When the device name in the fstab entry corresponds to a encryption device, the device
         # could not be found by that fstab name. In general, the encryptions might be probed with
-        # a different name, so before searching for the device, the devicegraph is modified to
-        # set the encryption names from the crypttab file. That changes are made in a temporary
+        # a different name, so before searching for the device, the encryption names from the crypttab
+        # file are saved into the proper encryption device. That changes are made in a temporary
         # devicegraph, so the original one is never altered.
         #
-        # @see #devicegraph_with_fixed_encryptions
+        # @see #devicegraph_with_crypttab_names
         #
         # @param entry [Y2Storage::SimpleEtcFstabEntry]
         # @param devicegraph [Y2Storage::Devicegraph]
         #
         # @return [Y2Storage::BlkDevice, Y2Storage::Filesystems::Nfs, nil] nil if the device is
         #   not found in the devicegraph.
         def entry_device(entry, devicegraph)
-          device = entry.device(devicegraph_with_fixed_encryptions(devicegraph))
+          device = entry.device(devicegraph_with_crypttab_names(devicegraph))
           device ? devicegraph.find_device(device.sid) : nil
         end
 
-        # Duplicates the given devicegraph and modifies it by setting the encryption names
-        # from the crypttab file
+        # Duplicates the given devicegraph and saves the encryption names from the crypttab file
         #
         # @see #crypttab
         #
         # @param devicegraph [Y2Storage::Devicegraph]
         # @return [Y2Storage::Devicegraph]
-        def devicegraph_with_fixed_encryptions(devicegraph)
+        def devicegraph_with_crypttab_names(devicegraph)
```

Comment:
Just thinking aloud: maybe now we don't need to duplicate the devicegraph now, since `save_crypttab_names` is less aggressive than `use_crypttab_names`.

Reply:
Umm, maybe, let me check if we have a new devicegraph each time that a fstab is imported.

Reply:
After checking it, I am not sure if it is a good idea to populate the current devicegraph. When importing a fstab, the crypttab names are related to the the filesystem that contains the fstab being imported. But that crypttab names are not related at all with the current devicegraph in which the fstab is going to be imported. So, having those crypttab names in the current devicegraph sounds a little bit weird to me.

Reply:
Good point.